backend/app/assets/stylesheets/spree/backend/components/_messages.scss

```diff
@@ -55,3 +55,29 @@
 
   &.error a { color: very-light($color-error, 10) }
 }
+
+.deprecation-warning {
```

Comment:
Formatting aside, @jhawthorn makes a good point here. If this is going to be a reusable thing then having it in this file makes sense. If it's specific to a particular page then these styles should live there.